docs/concepts/source-ip/source-ip.md

```diff
@@ -0,0 +1,300 @@
+---
+---
+
+{% capture overview %}
+Applications running in a Kubernetes cluster find and communicate with each
+other, and the outside world, through the Service abstraction. This document
+explains what happens to the source IP of packets sent to different types
+of Services, and how you can toggle this behavior according to your needs.
+{% endcapture %}
+
+{% capture body %}
+
+### Terminology
+
+Throughout this document, we will use the following terms:
+
+* [NAT](https://en.wikipedia.org/wiki/Network_address_translation): network address translation
+* [Source NAT](/docs/user-guide/services/#ips-and-vips): replacing the source IP on a packet, usually with a node's IP
+* [Destination NAT](/docs/user-guide/services/#ips-and-vips): replacing the destination IP on a packet, usually with a pod IP
+* [VIP](/docs/user-guide/services/#ips-and-vips): a virtual IP, such as the one assigned to every Kubernetes Service
+* [Kube-proxy](/docs/user-guide/services/#virtual-ips-and-service-proxies): a network daemon that orchestrates Service VIP management on every node
+
+### Prerequisites
+
+A working, Kubernetes 1.5 cluster is required for the examples in this document.
```

Comment:
"You must have a working Kubernetes 1.5 cluster to run the examples in this document."